src/Restaurant.js

```diff
@@ -1,4 +1,5 @@
 import React from "react";
+import React, { useState } from "react";
```

Comment:
This line is stopping the code compiling - could you make sure your website will run before submitting it?